proto/content_analysis/sdk/analysis.proto

```diff
@@ -18,6 +18,15 @@ enum AnalysisConnector {
   PRINT = 4;
 }
 
+message Handshake {
+  enum Status {
+    STATUS_UNKNOWN = 0;
+    SUCCESS = 1;
+    FAILURE = 2;
+  }
+  required Status status = 1;
```

Comment:
Consider sticking to "optional," see [go/required-fields-harmful](https://goto.google.com/required-fields-harmful) for explanation

Reply:
Thanks I will also remove them from [this PR as well](https://github.com/chromium/content_analysis_sdk/pull/7)